src/options/views/edit/index.vue

```diff
@@ -516,10 +515,13 @@ function setupSavePosition({ id: curWndId, tabs }) {
   }
 }
 
-.touch .edit {
-  // fixed/absolute doesn't work well with scroll in Firefox Android
-  position: static;
-  // larger than 100vh to force overflow so that the toolbar can be hidden in Firefox Android
+.touch body {
```

Comment:
Just to confirm, so now this applies to all pages in options?

Reply:
Yes. The issue is that if the page cannot scroll, the toolbar of Firefox on Android will not hide automatically and will cover part of the viewport. This happens to all pages in options. This hack should fix all full screen pages or modals.